storybook/figma.json

```diff
@@ -212,5 +212,8 @@
         "https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29528%3A588403&t=GUxMZEWcfRO7pXJb-1",
         "https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29528%3A588563&t=GUxMZEWcfRO7pXJb-1",
         "https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=29528%3A587660&t=GUxMZEWcfRO7pXJb-1"
+    ],
+    "UserAgreementPopup": [
+        "https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba⎜Desktop?type=design&node-id=1023-298171&t=Q4MOViPCoHsTjhs6-0"
```

Comment:
the link seems to be wrong/outdated

![Screenshot from 2023-06-16 14-30-31](https://github.com/status-im/status-desktop/assets/20650004/f80e21af-75ae-434d-b07c-82cf14537197)